### PR TITLE
fix `xarray_image` `to_zarr` if `xr.DataArray`

### LINKED ISSUE #122: DataArray object has no attribute 'to_zarr'
I have a catalog that contains images in jpg format.

```
images:
    driver: xarray_image
    description: "Images in jpg format"
    args:
      urlpath: "s3://path/to/*.jpg"
      storage_options:
        key: "{{env(AWS_KEY)}}"
        secret: "{{env(AWS_SECRET)}}"
        client_kwargs:
          endpoint_url: "https://s3.endpoint.com"
```

But when I try to export using `catalog.images.export('./catalogs')` I'm facing the following error:

```
/tmp/ipykernel_34689/1348189355.py in <cell line: 1>()
----> 1 catalog.images.export('./catalogs')

~/miniconda3/envs/test/lib/python3.8/site-packages/intake/source/base.py in export(self, path, **kwargs)
    401         representation (``.yaml()``).
    402         """
--> 403         return self._export(path, **kwargs)
    404 
    405     def _export(self, path, **kwargs):

~/miniconda3/envs/test/lib/python3.8/site-packages/intake/source/base.py in _export(self, path, **kwargs)
    409         method = container_map[self.container]._persist
    410         # may need to create path - access file-system method
--> 411         out = method(self, path=path, **kwargs)
    412         out.description = self.description
    413         metadata = {'timestamp': time.time(),

~/miniconda3/envs/test/lib/python3.8/site-packages/intake_xarray/xarray_container.py in _persist(source, path, **kwargs)
    156         """
    157         from intake_xarray import ZarrSource
--> 158         source.to_dask().to_zarr(path, **kwargs)
    159         return ZarrSource(path)
    160 

~/miniconda3/envs/test/lib/python3.8/site-packages/xarray/core/common.py in __getattr__(self, name)
    237                 with suppress(KeyError):
    238                     return source[name]
--> 239         raise AttributeError(
    240             f"{type(self).__name__!r} object has no attribute {name!r}"
    241         )

AttributeError: 'DataArray' object has no attribute 'to_zarr'
```

### PR DESCRIPTION
**Pull request by aaronspring**
fix #122

### CODE REVIEW

**Comment by aaronspring** [comment]
just upstream fails due to this notorious rasterio import error

File: intake_xarray/tests/test_image.py [hunk 1]
```diff
@@ -184,3 +184,10 @@ def test_read_images_and_exif_as_glob_with_coerce():
     ds = source.read()
     assert ds['raster'].shape == (3, 256, 256, 3)
     assert ds['EXIF Image ImageWidth'].shape == (3,)
+
+
+def test_read_images_and_persist():
+    pytest.importorskip('skimage')
+    urlpath = os.path.join(here, 'data', 'images', '*')
+    source = ImageSource(urlpath=urlpath, coerce_shape=(256, 256))
+    source.export('./exported')
```
Comment (martindurant) [inline intake_xarray/tests/test_image.py:193]: better use a temporary directory, else these will pollute the local repo. Shouldn't we also load them again, to check that the export worked?

Comment (aaronspring) [reply intake_xarray/tests/test_image.py]: using `tempfile` now

Comment (aaronspring) [reply intake_xarray/tests/test_image.py]: how to load again? just with `xr.open_dataset(engine="zarr")` or with `intake`?

**Pull request merged**